src/operator/LimitsInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {FC} from 'react'
+import {Input, InputType} from '@influxdata/clockface'
+import {get} from 'lodash'
+import {set} from 'lodash/fp'
+import {OrgLimits} from 'src/types/operator'
+
+interface Props {
+  type: InputType
+  name: string
+  limits: OrgLimits
+  onChangeLimits: (limits: OrgLimits) => void
+}
+
+const LimitsInput: FC<Props> = ({type, name, limits, onChangeLimits}) => {
+  const onChange = e => {
+    const newValue =
+      type === InputType.Number ? parseFloat(e.target.value) : e.target.value
+    const newLimits = set(name, newValue, limits)
+    onChangeLimits(newLimits)
+  }
+  return (
+    <Input
+      type={type}
+      name={name}
+      value={get(limits, name, '')}
```

Comment:
I'm not familiar with the difference. FWIW the Quartz UI is using the generic `lodash` for the `get` and the `lodash/fp` for the set. Don't wanna mess with it too much if it's working :)

Reply:
Oh, weird, OK. I thought it was using `/fp` for both. `lodash/fp` is written so that the "thing" being operated on (the limits, in this case) is the last argument in the list. It allows you to do fancy things like currying and other functional programming things.